# Supervised-Wisconsin Diagnostic Breast Cancer (WDBC)

Diagnostico de cancer de piel a partir de descriptores obtenidos de imagenes de manchas y lunares

http://archive.ics.uci.edu/ml/datasets/Breast+Cancer+Wisconsin+%28Diagnostic%29

---
## Imports

In [4]:
import numpy as np
from sklearn.model_selection import train_test_split

## cargar BD y labels

In [6]:
X=np.loadtxt('data.txt')
y=np.loadtxt('labels.txt')
print(data.shape, labels.shape)

(560, 30) (560,)


Se tienen 560 muestras y 30 características extraidas de imagenes de lunares http://archive.ics.uci.edu/ml/machine-learning-databases/breast-cancer-wisconsin/wdbc.names

---
Las características incluyen


1. Radio del lunar
+ Textura
+ Perimetro
+ Area
+ Forma
+ Compactación
+ Concavidad
+ Puntos de concavidad
+ Simetria
+ Dimensión Fractal

La media, desviación, estándar, y máximo de estos 10 descritpreos es calculada, formando las 30 características
    




## Separación de datos en entrenamiento y test

In [7]:
X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2, random_state=42)


print(X_train.shape, X_test.shape, y_train.shape, y_test.shape)

(448, 30) (112, 30) (448,) (112,)
